pkgs/development/python-modules/caldav/default.nix

```diff
@@ -44,7 +45,7 @@ buildPythonPackage rec {
 
   nativeCheckInputs = [
     pytestCheckHook
-    xandikos
+    (xandikos.override { python3Packages = python.pkgs; })
```

Comment:
```suggestion
    (python.pkgs.toPythonModule (xandikos.override { python3Packages = python.pkgs; }))
```

Reply:
This seems to be a nop change?

Reply:
https://github.com/NixOS/nixpkgs/blob/6ca95ec3ada7039ef28be4681f0313cee024083a/pkgs/development/interpreters/python/python-packages-base.nix#L47